### DISCUSSION TITLE
About the secondary development of the project

### DISCUSSION THREAD

**Discussion by ShawnLeung87** [AltraMayor/gatekeeper#549] [Ideas]
I want to carry out secondary development of this project and add web cpanel
1、I want to develop data display for this project, from which api/C function interface do I want to get ip flow data for aggregate display
2、policy management，I read the function of the lua script for policy management, and found that the policy parameters are not clearly described. Is there a document explaining what parameters the policy needs to pass to the C function?

**Comment by AltraMayor** [comment]
1. For aggregate data, the best source is the log files. If you write a script to collect and output the aggregate data, we'd want to merge this script, so other users have it as a reference for obtaining aggregate data. Once the script is done, you can modify it to suit the needs of your project.

2. Could you add more information here? For example, which function are you talking about?

**Comment by andrenth** [comment]
Hello

We're working on a system to aggregate Gatekeeper log statistics and export them to InfluxDB. It will soon be available [in this repository](https://github.com/hostnetbr/gatekeeper-log-exporter).

**Reply by andrenth** [reply]
The code for the exporter has been pushed to the abovementioned repository.

**Reply by andrenth** [reply]
Please note that the exporter parses the log format that will be used in Gatekeeper 1.1.
